english/swami-vivekananda/teachings-of-swami-vivekananda/quotes.md

```diff
@@ -0,0 +1,39 @@
+﻿Shri Swami Vivekananda says:
+
+
+1. Education is not the amount of information that we put into your brain and runs riot there, undigested, all your life.

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.